Lib/idlelib/autocomplete_w.py

```diff
@@ -17,7 +17,7 @@
 # before the default specific IDLE function
 KEYPRESS_SEQUENCES = ("<Key>", "<Key-BackSpace>", "<Key-Return>", "<Key-Tab>",
                       "<Key-Up>", "<Key-Down>", "<Key-Home>", "<Key-End>",
-                      "<Key-Prior>", "<Key-Next>")
+                      "<Key-Prior>", "<Key-Next>", "<Key-Escape>")
```

Comment:
I agree that Escape should close the window, but this is a different and unrelated change that belongs on a separate issue/PR.  Revert.
```suggestion
                      "<Key-Prior>", "<Key-Next>")
```

Reply:
OK. Should I open a new one with just this change?